www/gatsby-node.js

```diff
@@ -1040,6 +1040,23 @@ exports.onCreateNode = ({ node, actions, getNode, reporter }) => {
           )
         })
     }
+  } else if (node.internal.type === `CreatorsYaml`) {
+    // Creator pages
+    const validTypes = {
+      individual: `people`,
+      agency: `agencies`,
+      company: `companies`,
+    }
+
+    if (!validTypes[node.type]) {
+      throw new Error(
+        `Creators must have a type of “individual”, “agency”, or “company”, but invalid type “${node.type}” was provided for ${node.name}.`
+      )
+    }
+    slug = `/creators/${validTypes[node.type]}/${slugify(node.name, {
+      lower: true,
+    })}`
+    createNodeField({ node, name: `slug`, value: slug })
   }
```

Comment:
this code block was removed in #19495 Remove creator showcase

we should remove it again?

Reply:
open new issue #19738